test/unit/services/notifications/NotificationServiceSpec.scala

```diff
@@ -18,22 +18,21 @@ package services.notifications
 
 import java.io.IOException
 
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.xml.NodeSeq
+
 import base.UnitSpec
 import models.Notification
 import org.mockito.ArgumentMatchers.{any, eq => meq}
 import org.mockito.Mockito.{reset, verify, when}
 import org.mockito.{ArgumentCaptor, InOrder, Mockito}
-import org.scalatest.BeforeAndAfterEach
 import org.scalatest.concurrent.ScalaFutures
 import repositories.NotificationsRepository
 import testdata.notifications.ExampleXmlAndNotificationDetailsPair.exampleNotification
 import testdata.notifications.NotificationsTestData._
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
-import scala.xml.NodeSeq
-
-class NotificationServiceSpec extends UnitSpec with BeforeAndAfterEach with ScalaFutures {
+class NotificationServiceSpec extends UnitSpec with ScalaFutures {
```

Comment:
Aren't `ScalaFutures` already part of `UnitSpec` ?

Reply:
It was already removed but not committed yet :smile: 
I'll update it with the next PR.